### ISSUE TITLE
man page

### ISSUE THREAD

**Issue by carlwgeorge** [mholt/caddy#1876]
### 1. What version of Caddy are you using (`caddy -version`)?
0.10.9

### 2. What are you trying to do?
`man caddy`

Having a man page available would be useful for quick reference on servers and would help comply with distro packaging guidelines.

**Comment by tobya** [comment]
Thank you for the request.  We can add it to the list, if anyone was willing to put something together that could be discussed that would be great.

**Comment by fakeid30** [comment]
@tobya I'm willing to write the man page.

**Comment by tobya** [comment]
@fakeid30 That's great, maybe post it here for comments when done.

**Comment by fakeid30** [comment]
@tobya Hmm, I need a bit of a guide as to how you want me to do it. Can we talk on email ?

**Comment by tobya** [comment]
I dont know much about man pages, but if you take a stab at it and read up about their format you could get a basic draft together.  Look at other man pages to give you an idea of the sort of information that might be useful to have in a man page.

**Comment by theodesp** [comment]
How about using https://pandoc.org/ to convert to man pages? Is that viable?

**Comment by tobya** [comment]
Looks promising. It would certainly make the arcane world of man pages a little easier!

**Comment by 4470** [comment]
[Uploading pandoc-2.9.2.1.tar.gz…]()
https://pandoc.org/

**Comment by tobya** [comment]
@4470 Thank you for your note.  Have you managed to work on the man page?

**Comment by francislavoie** [comment]
@carlwgeorge do you still consider this relevant for v2? Is `caddy help` enough?

**Comment by carlwgeorge** [comment]
Yes, it's absolutely still relevant.  Man pages are the de facto standard on Linux.  They are either required or recommended for distribution packaging guidelines.

* https://docs.fedoraproject.org/en-US/packaging-guidelines/#_manpages
* https://www.debian.org/doc/debian-policy/ch-docs.html#manual-pages

Speaking for myself personally, when I need to check on the usage or flags of a command, I check the following resources in order.

1. `man <command>`
2. `<command> --help`
3. Google

**Comment by mholt** [comment]
Anyone is welcome to work on this. :)

Dunno if this'll help: https://godoc.org/within.website/x/internal/manpage
